### DISCUSSION TITLE
Play file at specific time

### DISCUSSION THREAD

**Discussion by kreativer-stuhlgang** [MiczFlor/RPi-Jukebox-RFID#1307] [Ideas]
I wanted to play a file at a certain time, e.g. a playlist or livestream in the morning. Is there already a bash script that can do the trick or would a cronjob be the right thing to start?

**Comment by MiczFlor** [comment]
Interesting... like an alarm clock :) ?
That's a new feature request.
Please create an "issue" under "issues" and give a couple of use scenarios, how this should work and why it would be useful, what problem it would solve. An "issue" we can place in a todo list. the discussions here would not be the best place to discuss feature requests.
Good idea, let's see what will come from it.
